source/extensions/transport_sockets/well_known_names.h

```diff
@@ -20,6 +20,7 @@ class TransportSocketNameValues {
   const std::string Tap = "envoy.transport_sockets.tap";
   const std::string Tls = "envoy.transport_sockets.tls";
   const std::string UpstreamProxyProtocol = "envoy.transport_sockets.upstream_proxy_protocol";
+  const std::string StartTls = "envoy.transport_sockets.starttls";
```

Comment:
We are trying to get rid of these files so just inline the strings where they are needed. Same below.

Reply:
What would say if I leave it as is and create a PR which removes well_known_names.h and inlines strings instead of variables across all files?

Reply:
OK that's fine.